mono/metadata/domain.c

```diff
@@ -1496,6 +1504,18 @@ mono_context_get (void)
 	return GET_APPCONTEXT ();
 }
 
+gint32
+mono_context_get_id (MonoAppContext *context)
```

Comment:
> The documentation should explain the lifecycle of those ids.

There's another precedent problem: The docs don't explain anywhere what the lifecycle of domain IDs is in the first place, for example. Happy to document if this is something we *want* to document.